src/crypto/rx-slow-hash.c

```diff
@@ -282,14 +318,9 @@ void rx_slow_hash(const uint64_t mainheight, const uint64_t seedheight, const ch
     memcpy(rx_sp->rs_hash, seedhash, HASH_SIZE);
   }
   if (rx_vm == NULL) {
-    randomx_flags flags = RANDOMX_FLAG_DEFAULT;
-    if (use_rx_jit()) {
-      flags |= RANDOMX_FLAG_JIT;
-      if (!miners)
-          flags |= RANDOMX_FLAG_SECURE;
+    if ((flags & RANDOMX_FLAG_JIT) && !miners) {
+        flags |= RANDOMX_FLAG_SECURE;
```

Comment:
Should this not be &'d against ~disabled flags ?

Reply:
Good point. Updated.